### DISCUSSION TITLE
Unsure if my setup is correct...

### DISCUSSION THREAD

**Discussion by ZeroFux78** [McCloudS/subgen#89] [Q&A] [answered]
I set up subgen in a docker container, enabled it as a provider in Bazarr, but I'm not sure if it's working correctly. 

Subgen logs: 
```
INFO:172.19.0.8:40956 - "POST /detect-language?encode=false HTTP/1.1" 200 OK
INFO:root:Bazarr-detect-language-gM2K0A is being handled handled by ASR.
INFO:faster_whisper:Processing audio with duration 00:30.000
INFO:faster_whisper:Detected language 'en' with probability 1.00
Detected Language: english
Transcribing with faster-whisper (medium)...

Transcribe:   0% 0/30.0 [00:00<?, ?sec/s]
Transcribe:  10% 3.12/30.0 [00:05<00:44,  1.66s/sec]
Transcribe:  82% 24.58/30.0 [00:12<00:02,  2.19sec/s]
Transcribe: 100% 30.0/30.0 [00:12<00:00,  2.43sec/s] 

Adjustment:   0% 0/29.98 [00:00<?, ?sec/s]
Adjustment: 100% 29.98/29.98 [00:00<00:00, 21450.91sec/s]
```

I have debug set to True, but thats all the info that I can see.  I don't know what file it's checking, but I haven't see any "aa" subtitles in my library (as far as I can tell)...  

So from the notes I'm seeing that it's successfully determining the language is english within the first 30 seconds, but I don't see anything after that.   I have **Search Enabled Providers Simultaneously** enabled in Bazarr, should I disable that or delete my other providers?

**Comment by McCloudS** [comment]
The AA language is only if subgen handles the naming, but you’re using bazarr, so it’ll be following its naming convention (likely en).  If the detect-language is working, the transcription ought to work if it’s called.  You can try manually searching a file and choosing whisper as the provider to force it.  I’m unsure of the nuances of the setting you mentioned.

Whisper will only run if it meets the cutoff and nothing else scores higher.  My guess is your simultaneous searching is finding something else scoring higher.

**Reply by ZeroFux78** [reply]
I hope I'm not being dense.... I've disabled the **Search Enabled Provider Simultaneously** option.  I don't see where I can manually search Whisper via Bazarr?  Is that a command line argument I need to do from the terminal window of the subgen docker?

Also, is there a guide if I wanted to do a plex webhook instead of bazarr?  I know the instructions say the token is needed, but I don't know the format of the webhook with the plex token.  A webhook as `http://subgen:9000` does post back to my subgen docker, but it gets a '405 Method Not Allowed' which is expected but I don't know how to fix.

**Reply by McCloudS** [reply, marked as answer]
If you navigate to a show and click the little person looking dude you can manually search and then pick your provider from the list.
![image](https://github.com/McCloudS/subgen/assets/64094529/a35c6dac-3e4b-4827-acf8-a6eb2f35ed36)
![image](https://github.com/McCloudS/subgen/assets/64094529/6bc46342-7d86-42df-8c9c-636949c3a9a9)

Your plex webhook would be `http://subgenip:9000/plex` and then you need your plex token, which you can find with https://[support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/](https://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/) and you'll set it under `PLEXTOKEN`.  You also need to make sure `PLEXSERVER` is correctly pointing to your plex server.

**Reply by Mx4flav0r** [reply]
can it also do translation to dutch? and how do i install it on a mac m1? i used pip install --upgrade --force-reinstall faster-whisper git+https://github.com/jianfch/stable-ts.git but cant find the launcher.py

**Reply by McCloudS** [reply]
> can it also do translation to dutch? and how do i install it on a mac m1? i used pip install --upgrade --force-reinstall faster-whisper git+https://github.com/jianfch/stable-ts.git but cant find the launcher.py

Follow the installation instructions: https://github.com/McCloudS/subgen#standalonewithout-docker

You can transcribe dutch, but cannot translate to it.
